src/templates/post-list-by-category.js

```diff
@@ -19,7 +19,7 @@ export default function PostListByCategoryPage({data, pageContext}) {
       <div className={styles.posts}>
         {Posts}
       </div>
-      <Paginator currentPage={pageContext.currentPage} totalPages={pageContext.totalPages}/>
+      <Paginator currentPage={pageContext.currentPage} totalPages={pageContext.totalPages} subsection={pageContext.slug}/>
```

Comment:
We could try to get this path information from props

Reply:
Raised https://github.com/ertrzyiks/yummy/issues/56 to look into that specifically since it sounds like a change on an app level.